app/models/manageiq/providers/lenovo/physical_infra_manager/parsers/components/physical_chassis_parser.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'component_parser'
+
+module ManageIQ::Providers::Lenovo
+  module Parsers
+    class PhysicalChassisParser < ComponentParser
+      class << self
+        # parse a node object to a hash with physical chassis data
+        # @param [XClarityClient::Node] node - object containing physical chassis data
```

Comment:
I think that you will want to change the `@param` to its new name `chassi_hash` and add some comment about the `rack` param.